.travis.yml

```diff
@@ -34,20 +40,23 @@ script:
   - go install github.com/golang/mock/mockgen
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/golang/protobuf/proto/...
+  - go get -u github.com/golang/protobuf/protoc-gen-go/...
+  - go install github.com/golang/protobuf/protoc-gen-go
   - MOCKFILES=$(find . -name mock_\*.go)
   - rm ${MOCKFILES}
   - PBFILES=$(find . -name \*.pb.go)
   - rm ${PBFILES}
   - golint --set_exit_status ./...  # Run golint while generated files are missing
   - go vet ./...
-  - git checkout -- ${PBFILES}
+  - go generate -run="protoc" -x ./...
```

Comment:
Nope, because the generations get interspersed and some of the `mockgen`s fail due to missing files.  I'll add a comment.